src/annotation_processor.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: guide
+title: PDL Schema
+permalink: /annotation_processor
+excerpt: Documentation of using annotation processor in Rest.Li
+---
+
+# Annotation Processor
+This sections introduces a generic schema annotation processing tool in Rest.Li. Before reading this, it is recommended to be familiar with the concept of PathSpec in Rest.Li. [Insert link here]
```

Comment:
I notice that you strictly refer to Rest.li as "Rest.Li", I'd suggest doing a search-and-replace to change all instances of "Rest.Li" to "Rest.li"

Reply:
Yes you are right and I replaced them all now.